src/Diagnostics.ModelsAndUtils/Models/Rendering.cs

```diff
@@ -110,7 +134,16 @@ public enum RenderingType
         /// Data rendered as Email.
         /// </summary>
         Email,
-        Insights
+
+        /// <summary>
+        /// Data rendered as a basic insight
+        /// </summary>
+        Insights,
+
+        /// <summary>
+        /// Data rendered as a basic insight
```

Comment:
correct the summary :). Since this is in ModelsAndUtils, we can utilize these summary messages for documentation purposes